### DISCUSSION TITLE
What am I missing? Gap problem.

### DISCUSSION THREAD

**Discussion by Tomisslaw** [templaza/astroid-framework#864] [Q&A]
I played a bit with the default Two theme, and with the Home Version 5 style. I played with the Layout and changed the gap in between the Articles in the Content section. And now I can't get back to the default state, with no gap. Above is the edited part, and bellow is the original. 
Framework Version: 3.1.7
Template Version: 1.0.6

![image](https://github.com/user-attachments/assets/9476f92c-727b-4975-899f-d8473cb70570)

**Comment by Banuk1616** [comment]
I can confirm that, it's the same for me.
Framework Version: 3.1.7
Template Version: 1.0.6
